tests/test_metric_reffree.py

```diff
@@ -0,0 +1,77 @@
+import biotite.interface.rdkit as rdkit_interface
+import pytest
+from rdkit import Chem
+from peppr.metric_reffree import (
+    LigandValenceViolations,
+    _count_valence_violations,
+    _count_valence_violations_atomarray,
+)
+
+
+def _test_mols() -> dict[Chem.Mol, int]:
+    good_smiles = [
+        "C1=CC=CC=C1",
+        "C1=CC=CC=C1[H]",
+        "N(=O)=O",
+        "C=P(=O)O",
+        "O=Cl(=O)O",
+        "OC1C(COP(=O)(OP(=O)(OP(=O)(O)O)O)O)OC(C1O)n1cnc2c1ncnc2N",
+    ]
+
+    one_viol = [
+        "C[C@H](FC)C",
+        "CO(C)C",  # This probably shouldn't read as 0 violations, but it does currently.
```

Comment:
I think for now it is OK to have permissive edge cases, which are not counted as violations when they probably should. Independent of the implementation in `RDKit` it would also be hard to tell for me when some seemingly wild atomic charges are actually valid, i.e. I think there are not really strict simple-to-implement rules for these cases, are there?